appservice/src/deploy/deployZip.ts

```diff
@@ -44,6 +44,16 @@ export async function deployZip(client: SiteClient, fsPath: string, configuratio
             const kuduClient: KuduClient = await getKuduClient(client);
             await kuduClient.pushDeployment.zipPushDeploy(fs.createReadStream(zipFilePath), { isAsync: true });
             await waitForDeploymentToComplete(client, kuduClient);
+            if (asp && asp.sku && asp.sku.tier && asp.sku.tier.toLowerCase() === 'basic') {
+                const deployments: number = (await kuduClient.deployment.getDeployResults()).length;
+                // if this is the first deployment, implement a 10 second delay for apps in a basic plan due to long start times
```

Comment:
Can you add a link to the issue for this since it's only a workaround and should be removed eventually?